src/include/session.h

```diff
@@ -163,30 +163,31 @@ struct __wt_session_impl {
 	u_int	stat_bucket;		/* Statistics bucket offset */
 
 /* AUTOMATIC FLAG VALUE GENERATION START */
-#define	WT_SESSION_CACHE_CURSORS		0x000001u
-#define	WT_SESSION_CAN_WAIT			0x000002u
-#define	WT_SESSION_IGNORE_CACHE_SIZE		0x000004u
-#define	WT_SESSION_INTERNAL			0x000008u
-#define	WT_SESSION_LOCKED_CHECKPOINT		0x000010u
-#define	WT_SESSION_LOCKED_HANDLE_LIST_READ	0x000020u
-#define	WT_SESSION_LOCKED_HANDLE_LIST_WRITE	0x000040u
-#define	WT_SESSION_LOCKED_METADATA		0x000080u
-#define	WT_SESSION_LOCKED_PASS			0x000100u
-#define	WT_SESSION_LOCKED_SCHEMA		0x000200u
-#define	WT_SESSION_LOCKED_SLOT			0x000400u
-#define	WT_SESSION_LOCKED_TABLE_READ		0x000800u
-#define	WT_SESSION_LOCKED_TABLE_WRITE		0x001000u
-#define	WT_SESSION_LOCKED_TURTLE		0x002000u
-#define	WT_SESSION_LOGGING_INMEM		0x004000u
-#define	WT_SESSION_LOOKASIDE_CURSOR		0x008000u
-#define	WT_SESSION_NO_DATA_HANDLES		0x010000u
-#define	WT_SESSION_NO_LOGGING			0x020000u
-#define	WT_SESSION_NO_RECONCILE			0x040000u
-#define	WT_SESSION_NO_SCHEMA_LOCK		0x080000u
-#define	WT_SESSION_QUIET_CORRUPT_FILE		0x100000u
-#define	WT_SESSION_READ_WONT_NEED		0x200000u
-#define	WT_SESSION_SCHEMA_TXN			0x400000u
-#define	WT_SESSION_SERVER_ASYNC			0x800000u
+#define	WT_SESSION_CACHE_CURSORS		0x0000001u
+#define	WT_SESSION_CAN_WAIT			0x0000002u
+#define	WT_SESSION_IGNORE_CACHE_SIZE		0x0000004u
+#define	WT_SESSION_INTERNAL			0x0000008u
+#define	WT_SESSION_LOCKED_CHECKPOINT		0x0000010u
+#define	WT_SESSION_LOCKED_HANDLE_LIST_READ	0x0000020u
+#define	WT_SESSION_LOCKED_HANDLE_LIST_WRITE	0x0000040u
+#define	WT_SESSION_LOCKED_METADATA		0x0000080u
+#define	WT_SESSION_LOCKED_PASS			0x0000100u
+#define	WT_SESSION_LOCKED_SCHEMA		0x0000200u
+#define	WT_SESSION_LOCKED_SLOT			0x0000400u
+#define	WT_SESSION_LOCKED_TABLE_READ		0x0000800u
+#define	WT_SESSION_LOCKED_TABLE_WRITE		0x0001000u
+#define	WT_SESSION_LOCKED_TURTLE		0x0002000u
+#define	WT_SESSION_LOCKED_TXN_GLOBAL		0x0004000u
```

Comment:
For consistency, this should probably be called `WT_SESSION_LOCKED_TXN_GLOBAL_WRITE` -- since that's (more) unwieldy I think it would be fine to call it `WT_SESSION_LOCKED_TXN_WRITE`.